index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+
+
+
+<header>
+<nav>
+   
+       <p>9:41</p>
+       <div class="ul">
+        <img src="image/CellularConnection.png" alt=""> 
+       <img src="image/Wifi.png" alt=""> 
+        <img src="image/Battery.png" alt="">
+       </div>
+       <P class="img"><img src="image/Union.png" alt=""></P>
+
```

Comment:
As requested by the previous reviewer, these Icons are not part of the Figma template rather they just serve to show that the site is being viewed on a phone. Kindly remove them and leave the only the logo_name and the mobile menu_bar